src/SIS_slow_thermo.F90

```diff
@@ -1361,20 +1361,14 @@ subroutine SIS2_thermodynamics(IST, dt_slow, CS, OSS, FIA, IOF, G, US, IG)
                                     scale=US%RZ_T_to_kg_m2s*Idt_slow)
   if (CS%id_qflim>0) call post_data(CS%id_qflim, qflx_lim_ice, CS%diag)
   if (CS%id_qfres>0) call post_data(CS%id_qfres, qflx_res_ice, CS%diag)
-  if (CS%id_net_melt>0) call post_data(CS%id_net_melt, net_melt, CS%diag)
-  if (CS%id_CMOR_melt>0) call post_data(CS%id_CMOR_melt, net_melt, CS%diag)
+  if (CS%id_net_melt>0) call post_data(CS%id_net_melt, IOF%seaice_melt, CS%diag)
+  if (CS%id_CMOR_melt>0) call post_data(CS%id_CMOR_melt, IOF%seaice_melt, CS%diag)
 
   if (coupler_type_initialized(IOF%tr_flux_ocn_top)) &
     call coupler_type_send_data(IOF%tr_flux_ocn_top, CS%Time)
 
   call disable_SIS_averaging(CS%diag)
 
-  ! Combine the liquid precipitation with the net melt of ice and snow for
```

Comment:
I think a bug fix flag here will work to get the old diagnostics, but it would be ideal (in my opinion) if there was a way to save `sea ice melt` and `sea ice melt + lprec` from the same simulation.